src/containers/App.js

```diff
@@ -25,8 +26,8 @@ const App = props => (
   })}
   >
     <div className="electron-titlebar" />
-    <SettingsMenu hideButton={props.isMenuOpen} />
-    <StageMenu hideButton={props.isMenuOpen} />
+    <SettingsMenu hideButton />
```

Comment:
Is this attribute redundant?

Reply:
Yes, this was just trying to get the menu back in with minimal changes. But it wouldn't be tricky to clean that up a bit.